pkg/codegen/merge_schemas.go

```diff
@@ -53,10 +53,10 @@ func valueWithPropagatedRef(ref *openapi3.SchemaRef) (openapi3.Schema, error) {
 	}
 
 	pathParts := strings.Split(ref.Ref, "#")
-	if len(pathParts) != 2 {
```

Comment:
This appears to be an artificial limitation for `pathParts` to have exactly 2 parts.